jep/1/README.adoc

```diff
@@ -0,0 +1,754 @@
+
+The final authority for JEP resolution is the BDFL. However, whenever a new
+JEP is put forward, any core developer that believes they are suitably
+experienced to make the final decision on that JEP may offer to serve as
+the BDFL's delegate (or "JEP czar") for that JEP. If their self-nomination
+is accepted by the other core contributors and the BDFL, then they will have
+the authority to accept (or reject) that JEP. This process happens most
+frequently with JEPs where the BDFL has agreed in principle that
+*something* needs to be done, but there are details that need to be worked out
+before the JEP can be accepted.
+
+If the final decision on a JEP is to be made by a delegate rather than
+directly by the BDFL, this will be recorded by including the
+"BDFL-Delegate" header in the JEP.
+
+JEP review and resolution may also occur on a list other than jenkinsci-dev@ In
+this case, the "Discussions-To" heading in the JEP will identify the
+appropriate alternative list where discussion, review and pronouncement on the
+JEP will occur.
+
+[[accepted]]
+==== Accepting a JEP
+
+For a JEP to be "Accepted" it must meet certain minimum criteria:
+
+* It must be a clear and complete description of the proposed enhancement.
+* The enhancement must represent a net improvement.
+* The proposed implementation, if applicable, must be solid and must not complicate Jenkins unduly.
+
+Once a JEP has been accepted, the implementation must be completed.
+The Jenkins project values contribution over "talk"
+footnote:[https://jenkins.io/project/governance/#meritocracy], and as such the
+implementation is of utmost importance to moving any proposal (Standards or
+Process) forward.
+
+[[final]]
+==== Finalizing a JEP
+When the implementation is complete and incorporated into the
+appropriate "main" code repository, the JEP author will change
+the JEP's status changed to "Final".
+
+[[maintenance]]
+==== JEP Maintenance
+
+Even after a JEP reaches "Final" status, it may need to be updated.
+
+In general, Standards track JEPs are not modified after they have
+reached the Final state. Once a Standards JEP has been completed, Jenkins developer
+documentation must become the formal documentation of the expected behavior.
+
+Informational and Process JEPs may be updated over time to reflect changes
+to development practices and other details. The precise process followed in
+these cases will depend on the nature and purpose of the JEP being updated.
+
+Replaced::  [[replaced]]
+Final JEPs may eventually also be "Replaced" - superseded by a different JEP -
+rendering the original obsolete.
+This is intended for Informational JEPs, where version 2 of an API can replace version 1.
+
+==== Other JEP Outcomes
+
+Not all JEPs will be accepted and finalized.
+
+Rejected::  [[rejected]]
+A JEP can also be "Rejected". Perhaps after all is said and done it
+was not a good idea. It is still important to have a record of this
+fact.
```

Comment:
Would be great to have an advice to the author about next steps. E.g. how to escalate or resubmit proposal

Reply:
Rejected is a pretty strong statement. I'm adding text to this effect.